x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/191_lookup_join_on_datastreams.yml

```diff
@@ -6,7 +6,7 @@ setup:
         - method: POST
           path: /_query
           parameters: []
-          capabilities: [enable_lookup_join_on_aliases]
+          capabilities: [enable_lookup_join_on_aliases_v2]
```

Comment:
Same here

Reply:
I'm confused - the test expectations are the same, why does this need a new cap?